docs/Measuring_Resonances.md

```diff
@@ -164,6 +191,62 @@ probe_points:
 It is advised to start with 1 probe point, in the middle of the print bed,
 slightly above it.
 
+#### Configure ADXL345 With Pi Pico
+
+##### Flash the Pico Firmware
+
+On your Raspberry Pi, compile the firmware for the Pico.
+
+```
+cd ~/klipper
+make clean
+make menuconfig
+```
+![Pico menuconfig](img/klipper_pico_menuconfig.png)
+
+Now, while holding down the `BOOTSEL` button on the Pico, connect the Pico to
+the Raspberry Pi via USB. Compile and flash the firmware.
+```
+make flash FLASH_DEVICE=first
+```
+
+If that fails, you will be told which `FLASH_DEVICE` to use. In this example,
+that's ```make flash FLASH_DEVICE=2e8a:0003```.
+![Determine flash device](img/flash_rp2040_FLASH_DEVICE.png)
+
+##### Configure the Connection
+
+The Pico will now reboot with the new firmware and should show up as a serial
+device. Find the pico serial device with `ls /dev/serial/by-id/*`. You can
+now add an `adxl.cfg` file with the following settings:
+
+```
+[mcu adxl]
+# Change <mySerial> to whatever you found above. For example,
+# usb-Klipper_rp2040_E661640843545B2E-if00
+serial: /dev/serial/by-id/usb-Klipper_rp2040_<mySerial>
+
+[adxl345]
+cs_pin: adxl:gpio1
+spi_bus: spi0a
+axes_map: x,z,y
+
+[resonance_tester]
+accel_chip: adxl345
+probe_points:
+    # Somewhere slightly above the middle of your print bed
+    147,154, 20
```

Comment:
There is one thing I forgot to mention (also on the thread in discourse)
Tests have shown that the 3V3 line on the Pico is very noisy, which leads to higher noise values on the ADXL measurements as well. This behavior can be improved by setting

```
[output_pin power_mode]
pin: adxl:gpio23
```

This will force the Pico's power source to go into high-performance mode rather than power saving mode.

Reply:
This (more or less) matches the configuration described [elsewhere in the document](https://www.klipper3d.org/Measuring_Resonances.html#configure-mpu-60009000-series-with-pico). I will go and test to see if the above works.

Reply:
True, it was added by an excellent documentation rework of @dingyifei but it applies to this as well

Reply:
Tested and change pushed. Seems to at least do no harm and, as you point out, may meaningfully improve things.